Makefile

```diff
@@ -0,0 +1,58 @@
+generator:=csharp
+openapi-generator-cli:=docker run --user $(shell id -u):$(shell id -g) --rm -v ${PWD}:/local -w /local openapitools/openapi-generator-cli:v6.2.1
```

Comment:
Please use the JAR based installation instead of Docker, it's easier to use. See https://github.com/Adyen/adyen-java-api-library/blob/b8ff6ef0f9f49e0f0ed3d6986bb72fb68f527303/Makefile#L3.